docs/slurk_deployment.rst

```diff
@@ -11,16 +11,16 @@ slurk reads a few environment variables for configuration. Because slurk is a fl
 it reads ``FLASK_ENV`` and defaults to a more verbose logging, when ``FLASK_ENV=development``.
 Also, the admin token is fixed to ``00000000-0000-0000-0000-000000000000`` and for RapiDoc,
 the token is provided. To provide cookies, a secret key is required, which can be set with
-``SLURK_SECRET_KEY``, which defaults to a random value. This implies, that every time, the
+``SLURK_SECRET_KEY``, which defaults to a random value. This implies that every time the
 server is restarted, the cookies will be invalidated.
 
 slurk is backed by a database. When no database is provided, an in-memory database is used.
-This can be changed with setting ``SLURK_DATABASE_URI``. For a list of possible engines see
+This can be changed by setting ``SLURK_DATABASE_URI``. For a list of possible engines, see
 `the SQLAlchemy documentation <https://docs.sqlalchemy.org/en/14/core/engines.html#database-urls>`_,
 however only SQLite and Postgres are tested. If you notice errors with other providers, please
-feel free to `file an issue <https://github.com/clp-research/slurk/issues/new>`_. Also note, that
-SQLite behaves weirdly when using auto-increment keys, so deletion of entries may result in the
-same id being used again. Later, we will see, how slurk can be set up with a Postgres-database and
+`file an issue <https://github.com/clp-research/slurk/issues/new>`_. Also note, that
+SQLite behaves weirdly when using auto-increment keys, so deletion of entries may cause the
```

Comment:
I liked the earlier version more 🤔 or ``may cause the same id to be used again``